xchange-mexbt/src/main/java/com/xeiam/xchange/mexbt/dto/account/MeXBTWithdrawRequest.java

```diff
@@ -0,0 +1,29 @@
+package com.xeiam.xchange.mexbt.dto.account;
+
+import java.math.BigDecimal;
+
+import si.mazi.rescu.SynchronizedValueFactory;
+
+import com.xeiam.xchange.mexbt.dto.MeXBTInsRequest;
+import com.xeiam.xchange.mexbt.service.MeXBTDigest;
+
+public class MeXBTWithdrawRequest extends MeXBTInsRequest {
+
+  private final BigDecimal amount;
+  private final String sendToAddress;
+
+  public MeXBTWithdrawRequest(String apiKey, SynchronizedValueFactory<Long> nonceFactory, MeXBTDigest meXBTDigest, String ins, BigDecimal amount, String sendToAddress) {
+    super(apiKey, nonceFactory, meXBTDigest, ins);
+    this.amount = amount;
+    this.sendToAddress = sendToAddress;
+  }
+
+  public BigDecimal getAmount() {
```

Comment:
Currently, the request body is like:
00:28:24.677 [default] [main] TRACE si.mazi.rescu.HttpTemplate - Request body = ```{"amount":0.000000001} ```

Do you mean it should be like:
```{"amount":"0.000000001"} ```